Dockerfile

```diff
@@ -24,6 +25,13 @@ COPY --from=build-image /usr/local/lib /usr/local/lib
 COPY --from=build-image /usr/local/bin/lnls-get-n-unpack /usr/local/bin/lnls-get-n-unpack
 RUN lnls-get-n-unpack -r $RUNTIME_TAR_PACKAGES && \
     ldconfig
+RUN if [ -n "$RUNTIME_PIP_PACKAGES" ]; then \
+        apt update && \
+        apt install -y --no-install-recommends pip && \
+        pip install $RUNTIME_PIP_PACKAGES; \
+    apt clean && \
+    rm -rf /var/lib/apt/lists/*
+    fi
```

Comment:
What do you think about adding a line like `$([ -n "$RUNTIME_PIP_PACKAGES" ] && echo pip)` to the package list above?

Reply:
To explain this, I'd rather not repeat the commands for apt.